staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/connection_test.go

```diff
@@ -323,6 +323,9 @@ func TestConnectionRemoveStreams(t *testing.T) {
 	// remove all existing
 	c.RemoveStreams(stream0, stream1)
 
+	// remove nil stream should not crash
+	c.RemoveStreams(nil)
```

Comment:
This is fine, but I'd be more confident with a test that reproduces the actual crash, that way we can be confident in the diagnosis and not just the fix.